bin/install

```diff
@@ -42,6 +42,15 @@ install_crystal() {
     curl --silent --fail --location --create-dirs --output "$source_path" "$download_url" || fail "Could not download Crystal $version"
     tar zxf "$source_path" -C "$install_path" --strip-components=1
     rm -rf "$tmp_download_dir"
+
+    if [ "$platform" == "darwin" ]; then
+      # shellcheck disable=SC2155
+      local libyaml_path=$(otool -L "$install_path/embedded/bin/shards" | grep libyaml | cut -d " " -f1 | tr -d "\t")
```

Comment:
```
local libyaml_path
libyaml_path=...
```

Reply:
@smorimoto I've amended the original commit. See https://github.com/asdf-community/asdf-crystal/commit/0c64e6099bae169b4821ea4fde553619c9857fb9.